src/AstCreate.fs

```diff
@@ -522,3 +575,8 @@ type PreXmlDoc with
             if docs.IsSome
             then docs.Value
         ]
+
+type SynSimplePat with
+    static member CreateTyped(ident, ``type``) =
+        let ssp = SynSimplePat.Id(ident, None, false, false, false, range.Zero)
```

Comment:
Minor thing, probably better using a shortened version or a prime rather than double ticks, very unwieldy

Reply:
i.e. ``type``

Reply:
Yeah i should get out of the habit of using those. Since this was merged do you want another PR to fix it?